From fc0d0dca08a9290733deb7f775e66457af2ed667 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Mon, 31 Jul 2017 16:59:24 +0900
Subject: [PATCH 2755/2863] HADOOP-14677. mvn clean compile fails. Contributed
 by Andras Bokor.

(cherry picked from commit 0fd6d0f2d361536b2baf859ddbb082eb2eadcfcf)
(cherry picked from commit b19415f1a18f4a2a121408f777fd9eaaeb52d074)
(cherry picked from commit 55c4eb3cf4f5a2999d1b3f725d07f608c119c469)

Conflicts:
	hadoop-maven-plugins/pom.xml

Change-Id: I9db12733283f24fbc7e97caab3e956de00152031
---
 hadoop-maven-plugins/pom.xml |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hadoop-maven-plugins/pom.xml b/hadoop-maven-plugins/pom.xml
index 49e496c..de61d11 100644
--- a/hadoop-maven-plugins/pom.xml
+++ b/hadoop-maven-plugins/pom.xml
@@ -55,6 +55,9 @@
   <build>
     <plugins>
       <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+      </plugin>
+      <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-plugin-plugin</artifactId>
         <version>${maven.dependency.version}</version>
@@ -67,6 +70,7 @@
             <goals>
               <goal>descriptor</goal>
             </goals>
+            <phase>compile</phase>
           </execution>
         </executions>
       </plugin>
-- 
1.7.9.5

